lib/Field/styles.scss

```diff
@@ -464,4 +464,19 @@ $field-data-p-line-height: 1.688rem !default;
       display: flex;
     }
   }
+
+  .repeat-icon {
+    opacity: 0;
+    transition: opacity $animation-time-micro $animation-curve;
+  }
+  &:hover {
+    .repeat-icon {
+      opacity: 1;
+    }
+  }
+  &.field-active {
+    .repeat-icon {
+      opacity: 1;
+    }
+  }
```

Comment:
💚 What forced your hand into using SCSS over tailwind at this point? Was it the animations?

If so, it may be worth dropping them to keep everything in tailwind 🤔

Reply:
Fair enough, I've re-written this piece to use tailwind. My reason was because the Validations and Actions (these are NOT new components, they are used by both Field and FieldNew) are written with this pattern in mind, and for the sake of not rewriting them, I thought it would be consistent to keep it in SCSS, but I do prefer to have it in tailwind. I think we'll just treat Validations and Actions SCSS as tech debt, for another PR 👍